aws_lambda_builders/workflows/ruby_bundler/utils.py

```diff
@@ -0,0 +1,40 @@
+"""
+Commonly used utilities
+"""
+
+import os
+import platform
+import tarfile
+import subprocess
+
+
+class OSUtils(object):
```

Comment:
Eventually we want to move this to a shared library, so all that workflows will have access to it. But not blocking on this right now.

Reply:
Totally agreed, I will fully admit I'm copying for convenience and 100% support merging to a shared library. Didn't want to do it myself since I'm tentative on my Python skills and don't want to step on your own structure changes.